components/PriceFilter.tsx

```diff
@@ -1,17 +1,13 @@
-import React, { useEffect, useState } from 'react';
+import { useState } from 'react';
 
 interface PriceFilterProps {
-  handleApplyClick: (min: number, max: number ) => void;
+  handleApplyClick: (min: number, max: number) => void;
```

Comment:
함수의 의미가 조금 더 명확해도 좋을 것 같아요~

```suggestion
  handleApplyPriceFilterClick: (min: number, max: number) => void;
```